src/outputs/outputs_main.py

```diff
@@ -3,12 +3,20 @@
 import pandas as pd
 from datetime import datetime
 from typing import Callable, Dict, Any
+import toml
 
 import src.outputs.short_form_out as short
 import src.outputs.map_output_cols as map_o
 
 OutputMainLogger = logging.getLogger(__name__)
 
+# Get the shortform schema
+short_form_schema = toml.load("src/outputs/output_schemas/frozen_shortform_schema.toml")
+
+# Columns are needed for filtering
+short_form_cols = short_form_schema.keys()
```

Comment:
This code doesn't get called anywhere. Are these short form cols meant to be here?

Reply:
Fixed. It was orphan code